lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1116,7 +1115,9 @@ sub update_status {
         return $ret;
     }
 
-    $self->append_log($status->{log});
+    $self->append_log($status->{log},             "autoinst-log-live.txt");
+    $self->append_log($status->{serial_log},      "serial-terminal-live.txt");
+    $self->append_log($status->{serial_terminal}, "serial-terminal-live.txt");
```

Comment:
Yes. https://github.com/os-autoinst/openQA/pull/1019#pullrequestreview-9205660

It appears to work quite well.